bpf/lib/common.h

```diff
@@ -643,6 +643,36 @@ struct ipv4_revnat_entry {
 	__u16 rev_nat_index;
 };
 
+struct lb4_affinity_key {
+	// if netns_cookie=1, then it's a netns cookie, otherwise a client_ip
```

Comment:
nit: mix of `// style` and `/* style */` comments. please keep kernel style (latter) here